lib/settings-panel.js

```diff
@@ -324,6 +327,15 @@ export default class SettingsPanel extends CollapsibleSectionPanel {
       }
     } else if (type === 'array') {
       let arrayValue = (value || '').split(',')
+      arrayValue = arrayValue.reduce((values, val) => {
+        const last = values.length - 1
+        if (last >= 0 && values[last].endsWith('\\')) {
+          values[last] = values[last].replace(/\\$/, ',') + val
+        } else {
+          values.push(val)
+        }
+        return values
+      }, [])
```

Comment:
This logic is a bit convoluted, but I cannot think of a better way to do this without lookbehind regex support (I've just tried using lookbehind regexps on Atom and it seems like its Chromium version does not support them yet).

Maybe once Atom uses Electron 3 this can be refactored 😃